test/e2e-tests.sh

```diff
@@ -29,8 +29,7 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 
 # Script entry point.
 
-initialize $@
-
-# TODO(#8): Write E2E tests.
+echo "TODO(#8): Write E2E tests."
+# initialize $@
```

Comment:
Is this `initialize` still valid once a test is present? Is it harmful if we call `initialize $@` with no e2e tests at the moment?

Reply:
It's not harmful, but you're wasting time and resources creating a cluster that's not used. I'll add a comment clarifying.

Reply:
Actually I just removed the commented code.